MekHQ/src/mekhq/gui/dialog/AutoResolveBehaviorSettingsDialog.java

```diff
@@ -0,0 +1,560 @@
+        chooseLabel.setLabelFor(presetsList);
+        var headerPanel = new UIUtil.FixedYPanel();
+        headerPanel.add(chooseLabel);
+
+        presetsList.setSelectionMode(ListSelectionModel.SINGLE_SELECTION);
+        presetsList.addListSelectionListener(this);
+        presetsList.setCellRenderer(new PresetsRenderer());
+        presetsList.addMouseListener(presetsMouseListener);
+
+        result.add(headerPanel);
+        result.add(Box.createVerticalStrut(10));
+        result.add(presetsList);
+
+        return result;
+    }
+
+    private JPanel behaviorSection() {
+        JPanel result = new UIUtil.OptionPanel("BotConfigDialog.behaviorSection");
+        UIUtil.Content panContent = new UIUtil.Content();
+        panContent.setLayout(new BoxLayout(panContent, BoxLayout.PAGE_AXIS));
+        result.add(panContent);
+
+        panContent.add(buildSlider(braverySlidebar, Messages.getString("BotConfigDialog.braverySliderMin"),
+            Messages.getString("BotConfigDialog.braverySliderMax"),
+            Messages.getString("BotConfigDialog.braveryTooltip"),
+            Messages.getString("BotConfigDialog.braverySliderTitle")));
+        panContent.add(Box.createVerticalStrut(7));
+
+        panContent.add(
+            buildSlider(selfPreservationSlidebar, Messages.getString("BotConfigDialog.selfPreservationSliderMin"),
+                Messages.getString("BotConfigDialog.selfPreservationSliderMax"),
+                Messages.getString("BotConfigDialog.selfPreservationTooltip"),
+                Messages.getString("BotConfigDialog.selfPreservationSliderTitle")));
+        panContent.add(Box.createVerticalStrut(7));
+
+        panContent.add(buildSlider(aggressionSlidebar, Messages.getString("BotConfigDialog.aggressionSliderMin"),
+            Messages.getString("BotConfigDialog.aggressionSliderMax"),
+            Messages.getString("BotConfigDialog.aggressionTooltip"),
+            Messages.getString("BotConfigDialog.aggressionSliderTitle")));
+        panContent.add(Box.createVerticalStrut(7));
+
+        panContent.add(buildSlider(herdingSlidebar, Messages.getString("BotConfigDialog.herdingSliderMin"),
+            Messages.getString("BotConfigDialog.herdingSliderMax"),
+            Messages.getString("BotConfigDialog.herdingToolTip"),
+            Messages.getString("BotConfigDialog.herdingSliderTitle")));
+        panContent.add(Box.createVerticalStrut(7));
+
+        panContent.add(buildSlider(fallShameSlidebar, Messages.getString("BotConfigDialog.fallShameSliderMin"),
+            Messages.getString("BotConfigDialog.fallShameSliderMax"),
+            Messages.getString("BotConfigDialog.fallShameToolTip"),
+            Messages.getString("BotConfigDialog.fallShameSliderTitle")));
+
+        var buttonPanel = new JPanel(new FlowLayout(FlowLayout.RIGHT, 10, 10));
+        buttonPanel.setAlignmentX(SwingConstants.CENTER);
+        result.add(buttonPanel);
+
+//        savePreset.addActionListener(this);
```

Comment:
Do we need all of this commented out code?

Reply:
No no no, I will delete all of this